### ISSUE TITLE
Range configuration properties are not uniformly represented

### ISSUE THREAD

**Issue by ftsamis** [tardis-sn/tardis#624]
Currently, configuration properties of type quantity range, are represented in more than 3 ways throughout the configuration processing:
1. quantity_linspace object
2. {'start': ..., 'stop': .., 'num': ...} dict
3. {'start': ..., 'end': ..., 'num': ...} dict
4. {'start': ..., 'end': ..., 'bins': ...} dict
5. [<start>, <stop>, <num>] list

or even a combination of two of the above, like

``` python
 >>> config.spectrum
{'bins': 10000,
 'end': <Quantity 20000.0 Angstrom>,
 'frequency': <Quantity [  1.49896229e+14,  1.50480824e+14,  1.51065420e+14,...,
             5.99467997e+15,  5.99526456e+15,  5.99584916e+15] Hz>,
 'start': <Quantity 500.0 Angstrom>}
```

(in which case although there is a different unit in the 2 representations it is still redundant information)

TARDIS should only work with `quantity_linspace` objects for all the quantity range properties. Then if someone actually wants the start, stop or num values of the `quantity_linspace` he could easily access them by `start = spectrum[0]`, `stop = spectrum[-1]`, `num = len(spectrum)`.

**Update:** With #652 all ranges are stored in the `Configuration` objects as dicts of `{'start': ..., 'stop': .., 'num': ...}` (exactly as provided). The various `from_config` methods are free to do whatever they want with those dicts. All duplicate information have been removed from the `Configuration` object.

**Comment by unoebauer** [comment]
Status?

**Comment by ftsamis** [comment]
#652 removes duplicate information storing in the configuration object and stores the properties as provided. So, for quantities that would be a `{'start': ..., 'stop': .., 'num': ...}` dict.

This can be closed whenever #652 is merged.

**Comment by ftsamis** [comment]
Fixed as part of #652. Shift the linspace discussion in #679.
